assembl/static2/js/app/components/administration/exportSection.jsx

```diff
@@ -16,7 +16,7 @@ type Props = {
   annotation: string
 };
 
-class ExportSection extends React.Component<Object, Props, void> {
+class ExportSection extends React.Component<Props, void> {
   props: Props;
```

Comment:
With latest flow version, you can remove `props: Props;` and `state: State` on the class.